MAUI/Popup/popup-size.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Popup Size in MAUI Popup control | Syncfusion
+description: Learn here all about Popup Size support in Syncfusion MAUI Popup (SfPopup) control and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Popup Size in Maui Popup (SfPopup)
+
+SfPopup allows displaying the Popup at any desired width and height by setting the `SfPopup.WidthRequest` and `SfPopup.HeightRequest`. The Popup size can also be changed by setting width request and height request to the views loaded inside the templates of the Popup.
+
+In the following code snippet the Popup is loaded in 320x320 pixels, where width request is set for the ListView that is loaded as template content and height request is set for the `PopupView`.
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PopupMaui.HeightWidthPage"
+             xmlns:local="clr-namespace:PopupMaui"
+             xmlns:listView="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView"
+             xmlns:popup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup">
+    <ContentPage.BindingContext>
+        <local:ContactsViewModel x:Name="viewModel"/>
+    </ContentPage.BindingContext>
+    <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="heightWidth" Text="HeightWidth" Clicked="heightWidth_Clicked" WidthRequest="300"/>
+            <popup:SfPopup x:Name="popup" HeaderTitle="ListView" HeightRequest="320" ShowCloseButton="True" ShowFooter="True">
+                <popup:SfPopup.ContentTemplate>
+                    <DataTemplate>
+                        <listView:SfListView  x:Name="listView"  ScrollBarVisibility="Never" ItemSpacing="5" WidthRequest="320" 
+                          ItemsSource="{Binding contactsinfo}" 
+                          >
+                            <listView:SfListView.ItemTemplate>
+                                <DataTemplate>

```

Comment:
No need to show the full customization.
Highlight the Property setting (WidthRequest, HeightRequest

Reply:
Addressed the concerns provided.